subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -171,6 +171,15 @@ config BT_CTLR_ADV_DATA_BUF_MAX
 	  Maximum number of buffered Advertising Data payload across enabled
 	  advertising sets.
 
+config BT_CTLR_ADV_EXT_PDU_EXTRA_DATA
+	bool "Storage for extra data added to extended advertising PDU structure"
+	depends on BT_CTLR_ADV_EXT
+	help
+	  Enables possiblity to add extra data to extended advertising PDU
+	  structure that are used for preparation of advertising PDU. This gives
+	  possiblity to send additional information from ULL to LLL when it
+	  is already advertising.
```

Comment:
I've named the config as well as advertising PDU data structure member `extra_data` to make it general. At the moment there is only one use-case for it, to provide additional configuration for Radio that must be in sync with content of advertising PDU (more precisely CTEInfo field of ext. adv. header).  I wanted to be more generic, because it is just a memory transport information between ULL and LLL. It is no a part of actual PDU and may be used to provide any other data required by LLL. In future it may be used for any other purpose.

Name of a config is tightly related with a data structure where it is added. 

It is a good idea to make the config hidden, because it should not be altered by end user. It serves other functionalities.

Referring to description, have you got any suggestions how to make it easier to understand?

Reply:
If we make it hidden, I don't think we need to improve the text, as user's won't need to worry about it, and we can leave it as is. 

Regarding appending "_MEMORY" I don't think that really helps much either :)

Reply:
I've made the config to be hidden. Also I've decided to change the name with _MEMORY. If anyone has a better proposal for the name it is welcome. Also I've reworded description, hope it is better to understand the purpose of the config and memory related with it.

I decided to do changes. besides hide the option because it is still a documentation for other developers.